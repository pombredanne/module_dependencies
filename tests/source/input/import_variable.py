from nltk import corpus, stopwords, tokenize

output = tokenize.TextTilingTokenizer().tokenize(corpus.brown.raw()[0:10000])

output = [token for token in output if token not in stopwords]
