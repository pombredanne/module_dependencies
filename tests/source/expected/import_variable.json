{
    "dependencies": [
        "nltk.corpus.brown.raw",
        "nltk.stopwords",
        "nltk.tokenize.TextTilingTokenizer"
    ],
    "imports": [
        "nltk"
    ]
}
